arbnode/resource_management.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2023, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+
+package arbnode
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"regexp"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/node"
+	flag "github.com/spf13/pflag"
```

Comment:
Aliasing this package seems unnecessary

https://google.github.io/styleguide/go/decisions#import-renaming

Reply:
For whatever reason we had been aliasing it elsewhere in the code and I had just been copying it. I'll remove it here.

(Thanks for the in-depth review! I think with your help we can move towards writing more idiomatic go.)

Reply:
Because pflag is a drop-in replacement for flag, it's standard practice to alias it. They recommend it: https://github.com/spf13/pflag#usage

Reply:
Yes I think that would make sense to remove it here and we can clean up the rest as we touch those files.
(Glad to hear it's helpful!)

@PlasmaPower I think they just say if you alias it than all code will continue to function. It doesn't really say that you should.